server/CodegenSpec.ts

```diff
@@ -8,6 +8,7 @@ const $codegenEntry = $.taggedUnion("type", [
     $.field("metadataHash", $.sizedUint8Array(64)),
     $.field("chainName", $.str),
     $.optionalField("connection", $connectionSpec),
```

Comment:
This should no longer be optional
```suggestion
    $.field("connection", $connectionSpec),
```